src/api/test/fixtures/getConf.js

```diff
@@ -26,7 +29,9 @@ module.exports = () => {
         settleSpread: 0,
         starkTokenId:
           '0x180bef8ae3462e919489763b84dc1dc700c45a249dec4d1136814a639f2dd7b',
-        tokenAddress: '0x4c5f66596197a86fb30a2435e2ef4ddcb39342c9'
```

Comment:
Are these removed on the backend as well? (ideally we should keep them there for couple of iterations to ensure backwards compatibility)

Reply:
Totally aligned with you on that. Kept them in the backend. Will remove them on deployment week + 1